features/claims/advocate/scheme_eleven/advocate_supplementary_claim_submit.feature

```diff
@@ -23,22 +27,23 @@ Feature: Advocate tries to submit a supplementary claim for miscellaneous fees (
     Given I insert the VCR cassette 'features/claims/advocate/scheme_eleven/supplementary_fee_calculations'
 
     And I should see a page title "Enter defendant details for advocate supplementary fee claim"
+    Then the page should be accessible within "#content"
     When I click "Continue" in the claim form
     Then I should be in the 'Miscellaneous fees' form page
     And I should see the advocate categories 'Junior,Leading junior,QC'
     And the following miscellaneous fee checkboxes should exist:
-      | section | fee_description |
-      | miscellaneous | Confiscation hearings (half day uplift) |
-      | miscellaneous | Confiscation hearings (half day) |
+      | section       | fee_description                          |
```

Comment:
**not a blocker**, one reason for not tabbing these out is that in some editors the its more likely to cause wrapping and difficulty reading. but just saying.

Reply:
Applied by either my beautifier or editorconfig, I will try to configure my editor to accommodate this.